test/e2e/support/utils/new-post.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+import { visitAdmin } from './';
+
+export async function newPost( {
```

Comment:
Can we add JSDoc here? It wasn't provided but it might be very helpful in the long run. At some point we will need to generate docs based on those code comments to make util methods discovery easier.

Reply:
On next commit I will add JSDoc. Didn't wanted to clutter current commit, although it has quite few changes. But because there was mainly copy-pastying I think it's not that critical.

Reply:
We squash and merge at the end of the process so it will become a single commit. I'm not concerned that much as long as all e2e tests are green after refactoring is finished :)

Reply:
JSDoc added here:
https://github.com/WordPress/gutenberg/pull/11701/commits/568458ed23dbb3335c57aa0c0ad4cde8c20436d6